crates/bevy_ui/src/node_bundles.rs

```diff
@@ -71,7 +71,7 @@ pub struct ImageBundle {
     pub calculated_size: CalculatedSize,
     /// The background color, which serves as a "fill" for this node
     ///
-    /// When combined with `UiImage`, tints the provided image.
+    /// Combines with `UiImage` to tint the provided image.
```

Comment:
`UiImage` **need** a `BackgroundColor`, otherwise it won't render, I felt the previous phrasing implies it wasn't the case.